lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1693,7 +1693,7 @@ definitions:
         maxLength: 4
       sgi_management_iface_static_ip:
         type: string
-        format: ipv4
+        format: cidr
```

Comment:
even though this looks to be supported by the strfmt library, it's being generated as a flat string field with no validation, so you should add validation to this field inside the model's ValidateModel() method

Reply:
I want to get this in for the 1.2 release. since it is bit late, I would like to get this in as is. I can add validation as follow up.